packages/core/stories/segmented-button-group/segmented-button-group.qa.stories.tsx

```diff
@@ -38,7 +38,20 @@ export const Default: StoryFn<SegmentedButtonGroupProps> = () => {
 };
 
 Default.parameters = {
-  chromatic: { disableSnapshot: false },
+  chromatic: {
+    disableSnapshot: false,
+    modes: {
+      theme: {
+        themeNext: "disabled",
+      },
+      themeNext: {
```

Comment:
Now this is more obvious, I'll need to look into why this is not taken into effect

Reply:
Fixed it